### PR TITLE
ps2eps: add test

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: Library/Formula/ps2eps.rb [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
     bin.install "bin/ps2eps", "bbox"
     share.install "doc/man"
     doc.install "doc/pdf", "doc/html"
   end
+
+  test do
+    cp test_fixtures("test.ps"), testpath/"test.ps"
+    system bin/"ps2eps", testpath/"test.ps"
+    assert (testpath/"test.eps").exist?
+  end
```
Comment (mikemcquaid) [inline Library/Formula/ps2eps.rb]: :+1:

**Comment by xu-cheng** [comment]
En, test failed Mountain Lion only. And I don't know why. I also downloaded the bottle generated by Mountain Lion ci-bot. It works well on Yosemite. I don't have ML VM to debug with.

**Comment by mikemcquaid** [comment]
Hmm, not sure what the issue is here then.

**Comment by xu-cheng** [comment]
Finally, I fixed it. It's due to a missing shebang.

File: Library/Formula/ps2eps.rb [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
-  skip_clean 'bin' # Don't strip +x on ps2eps
+  depends_on "ghostscript"
 
   def install
-    system "#{ENV.cc} #{ENV.cflags} src/C/bbox.c -o bbox"
-    bin.install "bin/ps2eps", "bbox"
+    system ENV.cc, "src/C/bbox.c", "-o", "bbox"
+    bin.install "bbox"
+    # add missing shebang
+    (bin/"ps2eps").write("#!/usr/bin/env perl\n#{File.read("bin/ps2eps")}")
```
Comment (mikemcquaid) [inline Library/Formula/ps2eps.rb]: Could we write a wrapper script instead maybe? Also: can you report this upstream? Thanks!

**Comment by vszakats** [comment]
Both `url` and `homepage` supports https, it'd nice to update them.

File: Library/Formula/ps2eps.rb (hunk 2 above)
Comment (xu-cheng) [inline Library/Formula/ps2eps.rb]: Sent the email and not hearing back for a week. Maybe I should rewrite it as a wrapper script then merge it?

Comment (mikemcquaid) [inline Library/Formula/ps2eps.rb]: :+1:

**Pull request closed without merging**